app/src/main/res/values/dimens.xml

```diff
@@ -1,4 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="text_view_text_size">24sp</dimen>
+    <!--Onboarding dimens-->
+    <dimen name="onboarding_image_view_height">447dp</dimen>
+    <dimen name="onboarding_text_view_heading">32sp</dimen>
```

Comment:
The text sizes don't apply only to onboarding, but to more generic styleguides and they aren't only for TextViews, they can be sued for Buttons or input fields.
I'd suggest something like:

```